app/indexers/curation_concerns/work_indexer.rb

```diff
@@ -8,6 +8,7 @@ def generate_solr_document
         solr_doc[Solrizer.solr_name('member_ids', :symbol)] = object.member_ids
         Solrizer.set_field(solr_doc, 'generic_type', 'Work', :facetable)
         solr_doc[Solrizer.solr_name('suppressed', STORED_BOOL)] = object.suppressed?
+        solr_doc[Solrizer.solr_name('hasRelatedMediaFragment', :symbol)] = object.representative_id
```

Comment:
Are you sure that this isn't already being indexed by virtue of it being an ActiveFedora relationship?